tokoh.md

```diff
@@ -3,5 +3,7 @@
 - David J Malan
 - Steve Jobs
 - Bill Gates
+- Budi Rahardjo
```

Comment:
iya, dia adalah tokoh it terkenal asal indonesia

Reply:
terimakasih